mix.exs

```diff
@@ -45,7 +45,8 @@ defmodule Kadena.MixProject do
       {:credo, "~> 1.6", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false},
       {:ex_doc, "~> 0.24", only: :dev, runtime: false},
-      {:excoveralls, "~> 0.15", only: :test}
+      {:excoveralls, "~> 0.15", only: :test},
+      {:casex, "~> 0.4.2"}
```

Comment:
it is because we do not use camel cases in the maps, but Kadena's API tho.
with this we can transform %{network_id: ""} to a valid Kadena's API JSON which is %{networkId: ""}

Reply:
@santiagoboterokommit Yep, I got it ... my comment was focused on that you can solve it with plain elixir instead of using external libraries, please research if you can do it easily otherwise use the **casex** library.